src/components/SurahInfo/index.js

```diff
@@ -0,0 +1,84 @@
+import React, { Component, PropTypes } from 'react';
+import { connect } from 'react-redux';
+
+const style = require('./style.scss');
+
+@connect(
```

Comment:
sorry dude, I don't agree with this. what you need from the redux state can be passed in as props from `Surah/index.js`. So you should pass currentSurah as a prop and isShowingInfo. we want to reduce how we use `@connect` as much as possible.